glean-core/src/upload/directory.rs

```diff
@@ -56,9 +55,12 @@ impl PingDirectoryManager {
         }
     }
 
-    /// Get the pending pings directory path.
-    pub fn get_dir(&self) -> PathBuf {
-        self.data_path.join("pending_pings")
+    /// Get the paths to the directories that may contain ping files.
+    fn get_dirs(&self) -> [PathBuf; 2] {
+        [
+            self.data_path.join("pending_pings"),
```

Comment:
Let's add a comment about order being important here, since it will break tests otherwise.

Reply:
Is this still relevant after https://github.com/mozilla/glean/pull/717#discussion_r381227058?